avogadro/qtgui/meshgenerator.h

```diff
@@ -144,26 +220,24 @@ class AVOGADROQTGUI_EXPORT MeshGenerator : public QThread
   Vector3f m_stepSize;      /** The step size vector for cube. */
   Vector3f m_min;           /** The minimum point in the cube. */
   Vector3i m_dim;           /** The dimensions of the cube. */
-  Core::Array<Vector3f> m_vertices, m_normals;
-  Core::Array<unsigned int> m_indices;
+
+  Core::Array<Vector3f> points, normals;  
+  std::vector<gridEdge> gridEdges; // size (m_dim.y() * m_dim.z())
+  std::vector<unsigned char> cubeCases; //size ((m_dim.x() - 1) * (m_dim.y() - 1) * (m_dim.z() - 1))
+  std::vector<int> triCounter;  // size ((m_dim.y() - 1) * (m_dim.z() - 1))
+  std::vector<unsigned char> edgeCases; // size ((m_dim.x() - 1) * (m_dim.y()) * (m_dim.z()))
+  Core::Array<Vector3f> tris; // triangles of a mesh 
   int m_progmin;
   int m_progmax;
 
   /**
-   * These are the tables of constants for the marching cubes and tetrahedra
-   * algorithms. They are taken from the public domain source at
-   * http://local.wasp.uwa.edu.au/~pbourke/geometry/polygonise/
+   * These are the lookup tables for flying edges.
+   * Reference : https://github.com/sandialabs/miniIsosurface/blob/master/flyingEdges/util/MarchingCubesTables.h
    */
-  static const float a2fVertexOffset[8][3];
-  static const int a2iVertexOffset[8][3];
-  static const int a2iEdgeConnection[12][2];
-  static const float a2fEdgeDirection[12][3];
-  static const int a2iTetrahedronEdgeConnection[6][2];
-  static const int a2iTetrahedronsInACube[6][4];
-  static const long aiTetrahedronEdgeFlags[16];
-  static const int a2iTetrahedronTriangles[16][7];
-  static const long aiCubeEdgeFlags[256];
-  static const int a2iTriangleConnectionTable[256][16];
+  static const unsigned char numTris[256]; 
+  static const bool isCut[256][12];  
+  static const char caseTriangles[256][16];
+  static const unsigned char edgeVertices[12][2];     
```

Comment:
The static analysis says these class members are never used?
If they're used, please keep to `m_edgeVertices` etc. for consistency.